crates/binjs_generate_library/src/lib.rs

```diff
@@ -1110,9 +1112,11 @@ impl<R> InnerDeserialization<{rust_name}> for Deserializer<R> where R: TokenRead
         print_file_structure!(self.reader, \"{name} {{{{\");
 {fields_def}
         print_file_structure!(self.reader, \"}}}}\");
-        Ok({rust_name} {{
+{maybe_exit_scoped_dictionary}
+        let result = Ok({rust_name} {{
 {fields_use}
-        }})
+        }});
+        result
```

Comment:
is there any reason to assign to local variable here?

Reply:
Ah no, it's a leftover from a previous iteration. I'll remove that bit.